docs/mkdocs.yml

```diff
@@ -36,25 +36,28 @@ nav:
   - Quick Start: direct/get-started.md
   - User Guide:
       - Overview: direct/user-guide-intro.md
-      - Prerequisites: direct/pre-reqs.md
-      - Acquire cluster for KubeFlex Hosting: direct/acquire-hosting-cluster.md
-      - Initialize KubeFlex Hosting cluster: direct/init-hosting-cluster.md
-      - Inventory and Transport Spaces: direct/its.md
-      - Workload Description Spaces: direct/wds.md
-      - Workload Execution Clusters: direct/wec.md
-      - Register a Workload Execution Cluster: direct/wec-registration.md
-      - Control:
-          - Overview: direct/control.md
-          - API reference (new tab): https://pkg.go.dev/github.com/kubestellar/kubestellar/api/control/v1alpha1
-          - Binding: direct/binding.md
-          - Transforming desired state: direct/transforming.md
-          - Combining reported state: direct/combined-status.md
-      - Core Helm chart: direct/core-chart.md
-      - Quick Start: direct/get-started.md
-      - Example Scenarios: direct/example-scenarios.md
-      - Third-party integrations:
-          - ArgoCD to WDS: direct/argo-to-wds1.md
-      - Best practices: direct/best-practices.md
+      - Quickstart: direct/get-started.md
+      - Setup:
+          - Prerequisites: direct/pre-reqs.md
+          - Acquire cluster for KubeFlex Hosting: direct/acquire-hosting-cluster.md
+          - Initialize KubeFlex Hosting cluster: direct/init-hosting-cluster.md
+          - Inventory and Transport Spaces: direct/its.md
```

Comment:
I think the setup should include one simple option in a single link which includes 3 steps of creating spaces and registering wecs. 
All details should go to some other place: maybe another section in this menu "Setup configuration" or "Changing the default setup" or something ..

Reply:
Registering WECs but not creating them? Who needs this particular bundle? Note that the suggested bundle does not align with the core Helm chart.